src/context.rs

```diff
@@ -14,6 +13,7 @@ use crate::key::*;
 use crate::lot::Lot;
 use crate::message::*;
 use crate::param::Params;
+use crate::r#move::*;
```

Comment:
ugh, I didn't realize we had this name conflict, maybe we can just move this single function into anther module, avoiding the `r#` dance? cc @r10s

Reply:
no objections from my side. at some point in the past, dc_move.c was much more complicated and had a lot of code that justifies a separate file/module. 

actually, there is only one function left - so maybe just move it to context.rs?

Reply:
but any other place that fits and make things easier would also be fine for me :)